x509util/utils_test.go

```diff
@@ -176,6 +182,38 @@ func Test_generateSubjectKeyID(t *testing.T) {
 	}
 }
 
+func Test_generateSubjectKeyID_fips(t *testing.T) {
+	if !fipsutil.Enabled() {
+		t.Skip("FIPS 140-3 mode is not enabled")
+	}
```

Comment:
Could `t.Setenv` help executing the tests at all times, but with the `GODEBUG` env var set for this test?

Reply:
No, I've tried but GODEBUG cannot be changed after initializing, it will panic if you do it. See https://github.com/golang/go/blob/279da965329a74cd75320f15cb9672a282690ab7/src/crypto/fips140/fips140.go#L26-L28
```go
if currentlyEnabled != fips140.Enabled {
        panic("crypto/fips140: GODEBUG setting changed after program start")
}
```